scholarly_content_detail/error_handlers.py

```diff
@@ -0,0 +1,15 @@
+from flask import json
+
+
+def http_error_handler(error):
+    """Return JSON instead of HTML for HTTP errors."""
+    # start with the correct headers and status code from the error
+    response = error.get_response()
+    # replace the body with JSON
```

Comment:
would follow https://tools.ietf.org/html/rfc7807 for this error response, namely `type`, `title`, `detail` for the 3 fields respectively

Reply:
After having to link to three different documents for just the `title`; TL;DR.

Can you provide an example please?

Reply:
```
    response.data = json.dumps({
        "type": error.code,
        "title": error.name,
        "detail": error.description,
    })
```

Reply:
updated.